codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/PubCrateConstrainedCollectionGenerator.kt

```diff
@@ -105,38 +103,45 @@ class PubCrateConstrainedCollectionGenerator(
                     """
                     impl #{From}<#{Symbol}> for $name {
                         fn from(v: #{Symbol}) -> Self {
-                            ${ if (innerNeedsConstraining) {
+                            ${
+                    if (innerNeedsConstraining) {
                         "Self(v.into_iter().map(|item| item.into()).collect())"
                     } else {
                         "Self(v)"
-                    } }
+                    }
+                    }
```

Comment:
This is very unreadable, although it looks like you merely auto-formatted it. Suggest replacing these with writables.

Reply:
yeah @david-perez I don't think you have the ktlint pre-commit hook enabled

Reply:
It's running in CI; I shouldn't have been able to push anything that is not compliant with our formatting guidelines.